.golangci.yaml

```diff
@@ -24,6 +23,7 @@ linters:
     - golint
     - nosnakecase
     - testpackage
+    - gci
```

Comment:
Unnecessary and overkill considering we weren't applying anything custom, just the standard formatting. This is not removed in favor of `goimports` with the separation described in this PR description.